calyx-py/calyx/builder.py

```diff
@@ -632,6 +652,10 @@ def is_seq_mem_d1(self) -> bool:
         """Check if the cell is a SeqMemD1 cell."""
         return self.is_primitive("seq_mem_d1")
 
+    def name(self) -> str:
```

Comment:
Consider adding an `@property` decorator to this getter, so we can just do `thing.name` instead of `thing.name()`.